pages/store/[nftClassId]/index.vue

```diff
@@ -213,6 +213,32 @@
               </div>
             </div>
           </template>
+
+          <template #buyer-messages>
+            <div
+              v-for="buyer in buyerMessages"
+              :key="buyer.txHash"
+              class="p-4"
+            >
+              <div class="flex flex-col items-start gap-3">
+                <div class="flex items-center gap-2">
+                  <EntityItem
+                    :name="buyer.wallet"
+                    :wallet-address="buyer.wallet"
+                    :is-link-disabled="true"
+                  />
+                  <p
+                    class="text-gray-400 text-xs"
+                    v-text="new Date(buyer.timestamp).toLocaleString()"
+                  />
+                </div>
+                <p
+                  class="text-gray-900 whitespace-pre-wrap break-words"
```

Comment:
Please use Nuxt UI text color class instead of hardcoded color class

ref: https://ui3.nuxt.com/getting-started/theme#neutral